maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -434,7 +439,11 @@ class IOSDriver(
 
     override fun eraseText(charactersToErase: Int) {
         metrics.measured("operation", mapOf("command" to "eraseText")) {
-            runDeviceCall("eraseText") { iosDevice.eraseText(charactersToErase) }
+            try {
+                runDeviceCall("eraseText") { iosDevice.eraseText(charactersToErase) }
```

Comment:
these will quietly fail. How do we know if eraseText failed, if the exception is caught here?

Reply:
I've failed the command now instead instead of silently failing.